sw/airborne/firmwares/rotorcraft/stabilization/stabilization_attitude_heli_indi.c

```diff
@@ -0,0 +1,618 @@
+    notch_filter_set_filter_frequency(&actuator_notchfilter[INDI_ROLL], heli_indi_ctl.motor_rpm / 60.0f);
+    notch_filter_set_filter_frequency(&actuator_notchfilter[INDI_PITCH], heli_indi_ctl.motor_rpm / 60.0f);
+    notch_filter_set_filter_frequency(&actuator_notchfilter[INDI_YAW], heli_indi_ctl.motor_rpm / 60.0f);
+    notch_filter_set_filter_frequency(&actuator_notchfilter[INDI_THRUST], heli_indi_ctl.motor_rpm / 60.0f);
+    notch_filter_update(&actuator_notchfilter[INDI_ROLL], &_in[INDI_ROLL], &_out[INDI_ROLL]);
+    notch_filter_update(&actuator_notchfilter[INDI_PITCH], &_in[INDI_PITCH], &_out[INDI_PITCH]);
+    notch_filter_update(&actuator_notchfilter[INDI_YAW], &_in[INDI_YAW], &_out[INDI_YAW]);
+    notch_filter_update(&actuator_notchfilter[INDI_THRUST], &_in[INDI_THRUST], &_out[INDI_THRUST]);
+  } else {
+    _out[INDI_ROLL]   = _in[INDI_ROLL];
+    _out[INDI_PITCH]  = _in[INDI_PITCH];
+    _out[INDI_YAW]    = _in[INDI_YAW];
+    _out[INDI_THRUST] = _in[INDI_THRUST];
+  }
+}
+
+static inline void indi_apply_measurement_notch_filters(int32_t _out[], int32_t _in[])
+{
+  if (heli_indi_ctl.motor_rpm > INDI_NOTCH_MIN_RPM && heli_indi_ctl.enable_notch) {
+    notch_filter_set_filter_frequency(&measurement_notchfilter[INDI_ROLL], heli_indi_ctl.motor_rpm / 60.0f);
+    notch_filter_set_filter_frequency(&measurement_notchfilter[INDI_PITCH], heli_indi_ctl.motor_rpm / 60.0f);
+    notch_filter_set_filter_frequency(&measurement_notchfilter[INDI_YAW], heli_indi_ctl.motor_rpm / 60.0f);
+    notch_filter_set_filter_frequency(&measurement_notchfilter[INDI_THRUST], heli_indi_ctl.motor_rpm / 60.0f);
+    notch_filter_update(&measurement_notchfilter[INDI_ROLL], &_in[INDI_ROLL], &_out[INDI_ROLL]);
+    notch_filter_update(&measurement_notchfilter[INDI_PITCH], &_in[INDI_PITCH], &_out[INDI_PITCH]);
+    notch_filter_update(&measurement_notchfilter[INDI_YAW], &_in[INDI_YAW], &_out[INDI_YAW]);
+    notch_filter_update(&measurement_notchfilter[INDI_THRUST], &_in[INDI_THRUST], &_out[INDI_THRUST]);
+  } else {
+    _out[INDI_ROLL]   = _in[INDI_ROLL];
+    _out[INDI_PITCH]  = _in[INDI_PITCH];
+    _out[INDI_YAW]    = _in[INDI_YAW];
+    _out[INDI_THRUST] = _in[INDI_THRUST];
+  }
+}
+
+static inline void indi_apply_actuator_butterworth_filters(int32_t _out[], int32_t _in[])
+{
+  for (uint8_t i = 0; i < INDI_DOF; i++) {
+    _out[i] = update_butterworth_2_low_pass_int(&actuator_lowpass_filters[i], _in[i]);
+  }
+}
+
+static inline void indi_apply_measurement_butterworth_filters(int32_t _out[], int32_t _in[])
+{
+  for (uint8_t i = 0; i < INDI_DOF; i++) {
+    _out[i] = update_butterworth_2_low_pass_int(&measurement_lowpass_filters[i], _in[i]);
+  }
+}
+
+/**
+ * @brief stabilization_attitude_heli_indi_set_steadystate_pitch
+ * @param pitch neutral pitch angle [deg].
+ *
+ * Change the neutral pitch angle.
+ */
+void stabilization_attitude_heli_indi_set_steadystate_pitch(float pitch)
```

Comment:
I use it in combination with a settings file, but wanted to keep that for the next PR